### PR TITLE
Marshmallow 3.0.0 and Python 3.7 Support extended

### PR DESCRIPTION
**Pull request by den4uk**
Marshmallow 3.0.0rc6 - dropped `PY2` support, introduced `_compact.py` to take care of backward compatibility.
Marshmallow 3.0.0 - fixes for breaking changes applied.
Also included Python 3.7 to tox.

### CODE REVIEW

**Comment by artemnesterenko** [comment]
Really waiting for this to get merged

**Comment by den4uk** [comment]
Dear @klen, could you please consider this PR, which enables Marshmallow v3 to work with this library. Thank you.

**Comment by artemnesterenko** [comment]
Is there any progress on this?

File: marshmallow_peewee/_compat.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+"""
+Marshmallow-3 dropped support for Python-2.
```
[the remark this answers is not in the document]
Comment (den4uk) [reply marshmallow_peewee/_compat.py]: @sloria Is satisfactory and good to go?

**Comment by den4uk** [comment]
> Looks good to merge. cc @klen

@klen sorry to bump it, but it has been a while, and there is a demand for this to function with the latest Marshmallow...
Thanks

**Comment by mivade** [comment]
This would be really useful to get merged.

**Comment by den4uk** [comment]
> This would be really useful to get merged.

And also released to pypi :+1:

**Comment by klen** [comment]
Hello guys, sorry for the late response I was extremely busy.

I made a decision to remove compatibility with python<3 (since it's deprecated) and Marshmallow<3 (for the version of marshmallow please use marshmallow-peewee<3).

So from now the library only supports python3+ and marshmallow3+

**Comment by klen** [comment]
Marshmallow-Peewee v.3 has been uploaded to Pypi.

**Comment by klen** [comment]
Thank @den4uk for the PR and pinging me.

**Comment by mivade** [comment]
Thanks @klen!

**Pull request merged**